app/views/webhook_deliveries/show.html.haml

```diff
@@ -0,0 +1,42 @@
+%h1 Webhook Delivery Details
```

Comment:
Example of the layout of this page

<img width="854" alt="Screenshot 2024-10-08 at 4 11 01 PM" src="https://github.com/user-attachments/assets/1667c044-d311-468f-bc47-a2f47661acad">